core/cloudflow-crd/src/main/scala/akka/datap/crd/App.scala

```diff
@@ -46,6 +46,9 @@ object App {
 
   final val Scope = "Namespaced"
 
+  final val ProtocolVersionKey = "protocol-version"
+  final val ProtocolVersion = "7"
```

Comment:
6, not 7?

Reply:
(or update all to 7, is maybe better, since the way that the protocol version is checked has changed)

Reply:
done